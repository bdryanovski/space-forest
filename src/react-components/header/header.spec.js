import React from 'react'
import renderer from 'react-test-renderer'
import {Header1, Header2, Header3, Header4, Header5, Header6} from './index'

describe('Headers', () => {
  it('should Header 1 match default snapshot', () => {
    const header = renderer.create(<Header1>Hello</Header1>)
    expect(header.toJSON()).toMatchSnapshot()
  })
  it('should Header 2 match default snapshot', () => {
    const header = renderer.create(<Header2>Hello</Header2>)
    expect(header.toJSON()).toMatchSnapshot()
  })
  it('should Header 3 match default snapshot', () => {
    const header = renderer.create(<Header3>Hello</Header3>)
    expect(header.toJSON()).toMatchSnapshot()
  })
  it('should Header 4 match default snapshot', () => {
    const header = renderer.create(<Header4>Hello</Header4>)
    expect(header.toJSON()).toMatchSnapshot()
  })
  it('should Header 5 match default snapshot', () => {
    const header = renderer.create(<Header5>Hello</Header5>)
    expect(header.toJSON()).toMatchSnapshot()
  })
  it('should Header 6 match default snapshot', () => {
    const header = renderer.create(<Header6>Hello</Header6>)
    expect(header.toJSON()).toMatchSnapshot()
  })
})
