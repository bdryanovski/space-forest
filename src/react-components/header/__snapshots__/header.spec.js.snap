// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headers should Header 1 match default snapshot 1`] = `
<h1>
  Hello
</h1>
`;

exports[`Headers should Header 2 match default snapshot 1`] = `
<h2>
  Hello
</h2>
`;

exports[`Headers should Header 3 match default snapshot 1`] = `
<h3>
  Hello
</h3>
`;

exports[`Headers should Header 4 match default snapshot 1`] = `
<h4>
  Hello
</h4>
`;

exports[`Headers should Header 5 match default snapshot 1`] = `
<h5>
  Hello
</h5>
`;

exports[`Headers should Header 6 match default snapshot 1`] = `
<h6>
  Hello
</h6>
`;
